.github/PULL_REQUEST_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Feature addition
+about: Use this if your PR adds a new feature to EssentialsX.
+labels: 'type: enhancement'
+---
+
+<!-- EssentialsX feature submission guide
+
+NOTE: Failure to fill out this template properly may result in your PR being
+      delayed or ignored without warning.
+
+Don't put anything inside this block, as it will be hidden when you post your
+PR. Make sure to also not type between any arrows in the template, as this text
+will also be hidden.
+
+Want to discuss your PR before submitting it? Join the EssentialsX Development
+server: https://discord.gg/CUN7qVb
+
+If you are submitting a new feature, please follow the following steps:
+
+1.  Fill out the template in full.
+      This includes providing screenshots and a link to the original feature 
+      request. If there isn't an existing feature request, we strongly
+      recommend opening a new feature request BEFORE opening your PR to
+      implement it, as this allows us to review whether we're likely to accept
+      your feature in advance, and also allows us to discuss possible
+      implementations for the feature. If there is no associated feature
+      request, your PR may be delayed or rejected without warning.
+      
+      You can open a new feature request by following this link:
+      https://github.com/EssentialsX/Essentials/issues/new/choose
+
+2.  If you are fixing a performance issue, please use the "Bug fix" PR template
+      instead. This is better suited to performance issues.
+
+3.  Include a demonstration.
+      If you are adding commands, please provide screenshots and/or a video
+      demonstration of the feature. Similarly, if you are adding new API,
+      please include a link to example code that takes advantage of your
+      proposed API. This will aid us in reviewing PRs and speed up the process
+      significantly.
+
+-->
+
+### Information
+
+<!--
+    Replace #nnnn with the number of the original issue. If this PR implements
```

Comment:
```suggestion
    Replace #nnnn with the number of the original issue. If this PR is implementing
```

Reply:
I don't feel this reads any better than the existing text

Reply:
I think I like the original as well